include/seastar/http/client.hh

```diff
@@ -169,6 +170,7 @@ class client {
 public:
     using reply_handler = noncopyable_function<future<>(const reply&, input_stream<char>&& body)>;
     using retry_requests = bool_class<struct retry_requests_tag>;
+    using clock_type = lowres_clock;
```

Comment:
I don't like this name "clock_type". It's very unclear what this clock is used for. Future developers may start using it for the wrong things (e.g., if you want to measure a latency histogram, you may not want to use lowres_clock to do this).
I think you should name it "timeout_clock" instead.

Reply:
Fair point. I followed the rpc_clock_type "naming convention" for no real reason